### PR TITLE
feat(grug): structured event

### PR DESCRIPTION
**Pull request by larry0x**

### CODE REVIEW

**Comment by larry0x** [comment]
@Rhaki can you use this branch for your work? you can merge your changes into the `grug/event` branch.

**Comment by Rhaki** [comment]
Yeah no problem

File: indexer/sql/src/active_model.rs [hunk 1]
```diff
@@ -60,20 +60,7 @@ impl Models {
...
-                transaction_id: Set(transaction_id),
-                block_height: self.block.block_height.clone(),
-                created_at: self.block.created_at.clone(),
-                r#type: Set(event.r#type.clone()),
-                attributes: Set(serialized_attributes.into_inner()),
-            };
-
-            self.events.push(new_event);
-        }
+        // TODO: handle structured events
```
Comment (larry0x) [inline indexer/sql/src/active_model.rs:63]: cc @penso handle the new event

File: grug/testing/tests/indexing.rs [hunk 2]
```diff
@@ -9,6 +9,7 @@ use {
 };
 
 #[test]
+#[ignore = "reason"]
 fn index_block_with_nonblocking_indexer() {
     let denom = Denom::from_str("ugrug").unwrap();
```
Comment (larry0x) [inline grug/testing/tests/indexing.rs:14]: cc @penso update this test

**Pull request merged**